testing/activedirectory/README.md

```diff
@@ -0,0 +1,13 @@
+## Test ds-load ldap plugin with active directory
+
+1. Install active directory in windows server 2022 https://computingforgeeks.com/install-active-directory-domain-services-in-windows-server/
```

Comment:
Do we need to follow all 17 steps in that tutorial (including domain controller, netBIOS, and DNS)?

Reply:
I don't remember exactly but I think I fallowed all 17 steps.